packages/dart_frog_cli/lib/src/commands/dev/dev.dart

```diff
@@ -70,13 +70,22 @@ class DevCommand extends DartFrogCommand {
         _defaultDartVmServicePort;
```

Comment:
The changes in this file illustrate precisely what this PR does: the Dev server runner lifecycle exceeds the lifetime of the "start" method.